lib/assessment.md

```diff
@@ -21,15 +21,25 @@ Using this format will make it easier for us to grade you, so that we can get ev
 ## Question 1
 
 When we say that the web is a service, what does that mean? Explain the interaction in terms of clients and servers.
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+<!-- the web faciliated the communication between clients and servers.  For example when you access a webpage, you are using the web to create a copy of the webpage from a server to your browswer or computer.  If you have another users looking the same page in their browser/computer, the two of you are looking at different copy rather than the same file. -->
```

Comment:
What you've said here is true, but it's not really a great explanation of what it means for the web to be a service. If you were to talk to someone with no knowledge of web development, how would you explain it to them?